### ISSUE TITLE
why remove VolumeSchedulingStageLatency metrics from scheduler?

### ISSUE THREAD

**Issue by likakuli** [kubernetes/kubernetes#122947]
### What happened?

we need to know how much time cost in volume binding stage, there is an old metrics called `VolumeSchedulingStageLatency`. it has been deprecated since 1.19. 
Just want to know why? i have found related pr but don't know why to deprecated this metric. https://github.com/kubernetes/kubernetes/pull/92160

### What did you expect to happen?

add metrics for scheduler to show the time cost of volume binding

### How can we reproduce it (as minimally and precisely as possible)?

create a pod with csi, when pod scheduled successfully, there is no related metrics to show the time cost of volume binding

### Kubernetes version

<details>

```console
$ kubectl version
master
```

</details>

### Cloud provider

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by kerthcet** [comment]
Generally, you can use plugin_execution_duration_seconds instead. If you have other requirements, you can feedback here.

**Comment by likakuli** [comment]
I would like to have a metric that excludes the waiting time for CSI binding, providing a more accurate reflection of the scheduler's duration. CSI binding relies on controller, making the timing unpredictable. Additionally, the maximum bucket is set to 200ms, which is too small for the time consumed by external CSI operations.

there is an metrics earlier which deprecated since 1.19.

**Comment by kerthcet** [comment]
That's true, the currently `plugin_execution_duration_seconds` with last bucket being [~22ms, Inf), which is not quite fit for this situation, this is similar to permit plugin I think, it has another metric named `PermitWaitDuration` for measuring the waiting time.

So my idea is let's add a new metric named  `SchedulerWaitDuration`, with labels `plugin`, `extensionPoint` and `status`,  with buckets `metrics.ExponentialBuckets(0.001, 2, 15)` up to 65s. And deprecate the `PermitWaitDuration`.

Any suggestions cc @Huang-Wei  @logicalhan @msau42

**Comment by kerthcet** [comment]
By the way:
- PermitWaitDuration(alpha):
https://github.com/kubernetes/kubernetes/blob/2c5105e7b80f97ab2da0d47a6db6875aec38331c/pkg/scheduler/metrics/metrics.go#L182-L190

- PluginExecutionDuration:
https://github.com/kubernetes/kubernetes/blob/2c5105e7b80f97ab2da0d47a6db6875aec38331c/pkg/scheduler/metrics/metrics.go#L162-L172

**Comment by kerthcet** [comment]
/remove-lifecycle stale
cc @kubernetes/sig-scheduling-misc for feedbacks.

**Comment by alculquicondor** [comment]
Can we just change the buckets of the existing metrics?

**Comment by likakuli** [comment]
already use a workaround way.

histogram_quantile(0.99, sum(rate(volume_operation_total_seconds_bucket{job=~"kube-controller-manager",operation_name="provision"}[5m])) by (le,plugin_name))

histogram_quantile(0.99, sum(rate(storage_operation_duration_seconds_bucket{job=~"kube-controller-manager"}[5m])) by (le,volume_plugin, operation_name))

use these two metrics can show the main csi controller cost. combine with existing scheduling metrics can filter out slower scheduling event caused by csi controller.

**Comment by kerthcet** [comment]
/remove-lifecycle stale

Will revisit this later
